packages/imperative/src/Imperative.ts

```diff
@@ -678,6 +683,20 @@ export class Imperative {
             }
             rootCommand.children.push(CompleteProfilesGroupBuilder.getProfileGroup(allProfiles, this.log));
         }
+        const authConfigs: {[key: string]: ICommandProfileAuthConfig[]} = {};
+        if (loadedConfig.profiles != null) {
+            loadedConfig.profiles.forEach((profile) => {
+                if (profile.authConfig != null) {
+                    for (const requiredOption of ["user", "password", "tokenType", "tokenValue"]) {
```

Comment:
Should we also require that an authConfig object have "host" and "port" options defined?

Reply:
Maybe? I was under the impression it would based on the prompting that was added, but I could be wrong.

Reply:
Added,

Reply:
I think that this is the right move, since I think this is about whether those properties are defined in the profile, not about whether they are required, or have been supplied.